docs/developers/quickstart/postman-sdk.md

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
I don't think this should go under quickstart

quickstart contains info for devs deploying contracts, not using the SDK
Would be okay with taking away quickstart and changing to deploy your contracts or some other folder structure if you think it's better

Reply:
I'm OK with this plan:
-Rename 'quickstart' to 'Deploy a contract'
-Create an 'SDK' section

Alternatively, the 'quickstart' section stays, and within it, we create a section for 'Deploy a contract' and 'SDK'